packages/lpa-questionnaire-web-app/src/routes/index.js

```diff
@@ -34,6 +35,7 @@ router.use(extraConditionsRouter);
 router.use(interestedPartiesRouter);
 router.use(representationsRouter);
 router.use(notifyingPartiesRouter);
+router.use(healthSafetyRouter);
```

Comment:
This router addition doesn't appear to have an associated update to the unit tests

Reply:
Added. Bit worried that doesn't flag a fail.